website/docs/language/expressions/types.mdx

```diff
@@ -79,6 +79,35 @@ comma-separated sequence of values, like `["a", 15, true]`.
 List literals can be split into multiple lines for readability, but always
 require a comma between values. A comma after the final value is allowed,
 but not required. Values in a list can be arbitrary expressions.
+### Sets
+
+Accessing elements of a set directly by index is not supported because sets are unordered collections. However, you can convert a set to a list to access elements by index.
```

Comment:
```suggestion
Terraform does not support directly accessing elements of a set by index because sets are unordered collections. To access elements in a set by index, first convert the set to a list.
```